operator/config/samples/kustomization.yaml

```diff
@@ -4,4 +4,5 @@ resources:
 - loki_v1beta1_alertingrule.yaml
 - loki_v1beta1_recordingrule.yaml
 - loki_v1beta1_rulerconfig.yaml
+- loki_v1_lokistack.yaml
```

Comment:
Do we still need the `v1beta1` sample? It is not possible to create that version anymore after this change.